frigate/object_processing.py

```diff
@@ -844,6 +845,33 @@ def should_mqtt_snapshot(self, camera, obj: TrackedObject):
 
         return True
 
+    def should_mqtt_motion(self, camera, motion_boxes):
+        # publish if motion is currently being detected
+        if motion_boxes:
+            # only send True if motion hasn't been detected recently
+            if self.last_motion_updates.get(camera, 0) == 0:
+                self.client.publish(
+                    f"{self.topic_prefix}/{camera}/motion/detected",
+                    "ON",
+                    retain=False,
+                )
+
+            # always updated latest motion
+            self.last_motion_updates[camera] = int(time.time())
+        elif not motion_boxes and self.last_motion_updates.get(camera, 0) != 0:
+            mqtt_delay = self.config.cameras[camera].motion.mqtt_off_delay
+            now = int(time.time())
```

Comment:
In the rest of the codebase, I use `datetime.datetime.now().timestamp()` to get the current time. Let's use that here for consistency.